lms/djangoapps/course_home_api/progress/v1/views.py

```diff
@@ -107,17 +107,18 @@ def get(self, request, *args, **kwargs):
             BlocksAPITransformer(None, None, depth=3),
         ]
         course = get_course_with_access(request.user, 'load', course_key, check_if_enrolled=True)
-        course_blocks = get_course_blocks(request.user, course_usage_key, transformers, include_completion=True)
 
         enrollment_mode, _ = CourseEnrollment.enrollment_mode_for_user(request.user, course_key)
 
         course_grade = CourseGradeFactory().read(request.user, course)
         courseware_summary = course_grade.chapter_grades.values()
 
         data = {
-            'course_blocks': course_blocks,
             'courseware_summary': courseware_summary,
+            'credit_course_requirements': credit_course_requirements(course_key, request.user),
             'enrollment_mode': enrollment_mode,
+            'certificate_data': get_cert_data(request.user, course, enrollment_mode, course_grade),
+            'studio_url': get_studio_url(course, 'settings/grading'),
```

Comment:
don't forget to add all this stuff into the doc string too. Also, for now, please remove the credit_course_requirements (at least from the view if not the view and serializer)